game/src/game/components/VehicleComponent.h

```diff
@@ -64,5 +69,21 @@ class VehicleComponent final : public Component,
         return g_vehicle_;
     }
 
+    // Setters
+    inline void SetVehicleDataPath(const std::string& data_path)
+    {
+        g_vehicle_data_path_ = data_path;
+    }
```

Comment:
Can we remove this? I dont think this path should change between vehicles, and it's set as a constant in the cpp file